DigitalLearningSolutions.Web/Controllers/TrackingSystem/DelegateApprovalsController.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem

```

Comment:
Could we put this + views into a folder called "Delegates" as we'll have several delegate related controllers under the Tracking system?